doc/merge_split.rst

```diff
@@ -0,0 +1,19 @@
+*Merge and Split 
```

Comment:
We need to refer to this file in `index.rst` to include the documentation on our page